FirebaseCore/Sources/FIRHeartbeatLogger.m

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FirebaseCore/Sources/Private/FIRHeartbeatLogger.h"
+
+#if SWIFT_PACKAGE
+@import HeartbeatLogging;
+#else
```

Comment:
`check` is failing because of the module import but based on what I've learned today, this module import is actually ok because it's gated by `#if SWIFT_PACKAGE`

@paulb777  @ryanwilson

Reply:
All other checks in b4cd35e were green though, including core's `spm-cron` jobs.

Reply:
Adding the `spm-cron-workflow` (9979a7a) surfaced the Swift library related limitation for cocoapods:
```
 -> FirebaseCore (8.10.0)
    - ERROR | [OSX] unknown: Encountered an unknown error (The following Swift pods cannot yet be integrated as static libraries:
```